test/2.0/test-middleware-swagger-validator.js

```diff
@@ -208,50 +208,117 @@ describe('Swagger Validator Middleware v2.0', function () {
       });
     });
 
-    it('should return an error for invalid parameter values based on type/format', function () {
-      var argName = 'arg0';
-      var badValue = 'fake';
-      var testScenarios = [
-        {in: 'query', name: argName, type: 'boolean'},
-        {in: 'query', name: argName, type: 'integer'},
-        {in: 'query', name: argName, type: 'number'},
-        {in: 'query', name: argName, type: 'string', format: 'date'},
-        {in: 'query', name: argName, type: 'string', format: 'date-time'},
-        {in: 'query', name: argName, type: 'array', items: {type: 'integer'}}
-      ];
-
-      async.map(testScenarios, function (scenario, callback) {
-        var cPetStore = _.cloneDeep(petStoreJson);
-        var cScenario = _.cloneDeep(scenario);
-        var content = {arg0: scenario.type === 'array' ? [1, 'fake'] : badValue};
-        var expectedMessage;
-
-        cPetStore.paths['/pets/{id}'].get.parameters = [cScenario];
-
-        if (scenario.type === 'array') {
-          expectedMessage = 'Parameter (' + argName + ') at index 1 is not a valid integer: fake';
-        } else {
-          expectedMessage = 'Parameter (' + scenario.name + ') is not a valid ' +
-                              (_.isUndefined(scenario.format) ?
-                                 '' :
-                                 scenario.format + ' ') + scenario.type + ': ' + badValue;
-        }
+    it('should return an error for invalid parameter values based on type/format', function (done) {
+        var argName = 'arg0';
```

Comment:
The extra whitespace isn't desired but if you've already squashed commits, I'm not sure I mind enough not to merge the PR.